packages/fxa-content-server/tests/functional.js

```diff
@@ -76,6 +75,7 @@ module.exports = testsSettings.concat([
   //'tests/functional/sign_up_with_code.js',
   //'tests/functional/sync_v3_email_first.js',
   // 'tests/functional/reset_password.js',
+  // 'tests/functional/legal.js',
```

Comment:
Do we want to disable this test while we have not enabled this 100% in production?

Reply:
@vbudhram it will be enabled in #15218, I plan to rebase on this so that PR passes.

Reply:
@ashrivastava-qa I think we'll also need the cannot create account test commented out here, see what's failing in #15218

Reply:
@LZoog AFAIR, we don't have a 'cannot create account' test in the old suite. Which ones are you refrerring to from the failed tests?

Reply:
@ashrivastava-qa Ah apologies, I must have misremembered what I saw yesterday, I was thinking of `/clear`. Either way your PR LGTM, I can comment out anything else in my branch.